### DISCUSSION TITLE
How do I activate AntiMicroX?

### DISCUSSION THREAD

**Discussion by tomek3001** [AntiMicroX/antimicrox#484] [Q&A]
There is no place where activation is explained. I run AntiMicroX as an administrator, mapped my pad keys, then I run the game (Fall Guys/Epic Games) and yet the game still detects my A button as LT, RT as LB etc.

**Comment by pktiuk** [comment]
What do you mean by activation?  
Buttons mapped in AntiMicroX trigger selected actions (like pressing keyboard button), but it doesn't make gamepad actions invisible for other apps.
